tardis/tests/integration_tests/test_integration.py

```diff
@@ -28,6 +30,7 @@ def setup(self, request, reference, data_path, atomic_data_fname):
         self.name = data_path['setup_name']
 
         self.config_file = os.path.join(data_path['config_dirpath'], "config.yml")
+        self.config_dict = yaml.load(self.config_file)
```

Comment:
That's bad, don't do this. We modify the `YAMLLoader`.
Use `from tardis import load_yaml` instead.